packages/mask/src/extension/popups/pages/Wallet/ContractInteraction/index.tsx

```diff
@@ -283,7 +282,7 @@ const ContractInteraction = memo(() => {
     const tokenPrice = useTokenPrice(chainId, !isNativeTokenInteraction ? token?.address : undefined)
     const nativeTokenPrice = useNativeTokenPrice(nativeToken?.chainId)
     const tokenValueUSD = new BigNumber(tokenAmount)
-        .dividedBy(pow10(tokenDecimals ?? 0))
+        .shiftedBy(-(tokenDecimals ?? 0))
```

Comment:
Why there is a minus sign?

Reply:
see https://mikemcl.github.io/bignumber.js/#shift